scissors-sample/src/main/java/com/lyft/android/scissorssample/App.java

```diff
@@ -47,5 +50,10 @@ public void onCreate() {
                 .penaltyLog()
                 .penaltyDeath()
                 .build());
+
+        // If using Android-Universal-Image-Loader
+        // Create global configuration and initialize ImageLoader with default config
+        ImageLoaderConfiguration config = new ImageLoaderConfiguration.Builder(this).build();
+        ImageLoader.getInstance().init(config);
```

Comment:
As picasso is default please comment this out mentioned to enable it for Android-Universal-Image-Loader